app/Models/Artist.ts

```diff
@@ -0,0 +1,34 @@
+import { DateTime } from 'luxon'
+import { BaseModel, column } from '@ioc:Adonis/Lucid/Orm'
+
+export default class Artist extends BaseModel {
+  @column({ isPrimary: true })
+  public id: string
+
+  @column()
+  public name: string
+
+  @column()
+  public genre: string
+
+  @column()
+  public description: string
+
+  @column()
+  public links: string
+
+  @column()
+  public logo: string
+
+  @column()
+  public members: string
+
+  @column()
+  public musicLabel: string
+
```

Comment:
discography?

Reply:
ich glaube ich würde das erst einmal nicht mit rein nehmen und später hinzufügen.

Reply:
mh, weß nich

Reply:
discography würde ich dann auch als neue table anlegen